presentation-api/controlling-ua/support/stash.js

```diff
@@ -0,0 +1,82 @@
+(window => {
+  //
+  // Common Parameters and Functions for test cases which use a stash
+  //
+  window.message1 = '1st';
+  window.message2 = '2nd';
+  window.message3 = new Uint8Array([51, 114, 100]);      // "3rd"
+  window.message4 = new Uint8Array([52, 116, 104]);      // "4th"
+  window.message5 = new Uint8Array([108, 97, 115, 116]); // "last"
+
+  Uint8Array.prototype.toText = function() {
```

Comment:
I do not know if it is considered good practice to extend the prototype of native objects. Now, I suppose it does not matter too much here since that extension will only appear in code we control (and not in a library that others could use).